flask_restplus_patched/parameters.py

```diff
@@ -1,9 +1,14 @@
 # encoding: utf-8
 # pylint: disable=missing-docstring
-from six import itervalues
+import logging
 
+from six import itervalues
 from flask_marshmallow import Schema, base_fields
-from marshmallow import validate
+from marshmallow import validate, validates_schema
+from marshmallow.exceptions import ValidationError
+from app.extensions.api import http_exceptions
```

Comment:
I'm sorry about my "let's merge abort into parameters", I meant you to raise `NameError` (?) or `ValueError`, or anything like that and catch it in `commit_or_abort` context manager.